tests/wpt/web-platform-tests/html/semantics/scripting-1/the-script-element/execution-timing/029.html

```diff
@@ -25,13 +28,24 @@
         log( 'inline script #2' );
         var t = async_test()
 
-        function test() {
+        function final_test() {
+                // In a subsequent task, the JS URL should have executed.
                 assert_any(assert_array_equals, eventOrder, [
-                  ['inline script #1', 'end script #1', 'JS URL', 'inline script #2'],
-                  ['inline script #1', 'end script #1', 'inline script #2', 'JS URL']]);
+                  ['inline script #1', 'end script #1', 'beforeunload event', 'inline script #2', 'JS URL'],
```

Comment:
The order of the `JS URL` and `inline script #2` events have changed?

Reply:
The test, before these changes, assumed that `JS URL` happens in the same task as the one in which we initiate the navigation, while the spec says to queue a subsequent task for the `JS URL` part. So that part is now guaranteed to be last, and I've added the `beforeunload event` part since that will happen in the "first" task where the navigation "starts". 

So basically 'beforeunload event' has taken the place of `JS URL`, while 'inline script #2' could fire either before or after it, and `JS URL` is now guaranteed to require a subsequent task. 

See the discussion at https://github.com/whatwg/html/issues/3730#issuecomment-492071447

Reply:
Might be worth putting a link to that issue in the test?

Reply:
Ok, done.